### PR TITLE
Remove dependency on jsonpath

### PR DESCRIPTION
**Pull request by shehzan10**
Remove the use of `jsonpath` function and puts in substitute code.

### CODE REVIEW

File: lib/combineNodes.js [hunk 1]
```diff
@@ -36,51 +35,77 @@ function combineNodes(gltf) {
...
-                    0.0, 1.0, 0.0, 0.0,
-                    0.0, 0.0, 1.0, 0.0,
-                    0.0, 0.0, 0.0, 1.0
-                ],
-                extras : {
-                    _pipeline : {}
+
+    var scenes = gltf.scenes;
+    for(var sceneId in scenes) {
+        if(scenes.hasOwnProperty(sceneId)) {
```
Comment (lilleyse) [inline lib/combineNodes.js]: Throughout format as `for (` and `if (`.

File: lib/combineNodes.js [hunk 2]
```diff
@@ -36,51 +35,77 @@ function combineNodes(gltf) {
...
-    jp.apply(gltf, '$.animations[*].channels[*].target.id', function(nodeId) {
-        targets[nodeId] = true;
-        return nodeId;
-    });
+    var animations = gltf.animations;
+    for(var animationId in animations) {
+        if(animations.hasOwnProperty(animationId)) {
+            var channels = animations[animationId].channels;
+            for(var channelId in channels) {
+                if(channels.hasOwnProperty(channelId)) {
```
Comment (lilleyse) [inline lib/combineNodes.js]: `channels` is an array, so just use a regular for loop.

File: lib/combineNodes.js [hunk 3]
```diff
@@ -36,51 +35,77 @@ function combineNodes(gltf) {
...
+                    var nodeId = channels[channelId].target.id;
+                    targets[nodeId] = true;
+                }
+            }
+
```
Comment (lilleyse) [inline lib/combineNodes.js]: Remove empty line.

**Comment by shehzan10** [comment]
@lilleyse Fixed.

**Pull request merged**